tests/integration/test_remoted/test_agent_communication/data/configuration_templates/config_agent_pending_status.yaml

```diff
@@ -0,0 +1,10 @@
+- sections:
+    - section: remote
+      elements:
+        - connection:
+            value: 'secure'
+        - port:
+            value: 'PORT'
+        - protocol:
+            value: 'PROTOCOL'
+
```

Comment:
Add always a new line (without spaces) at the end of the files. Check all files.